### ISSUE TITLE
[ISSUE][CLOSED] incorrect path of cibot_rest_api.sh

### ISSUE THREAD

**Issue by leemgs** [leemgs/TAOS-CI8#138]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Wednesday Jul 18, 2018 at 07:16 GMT_
_Originally opened as https://github.sec.samsung.net/STAR/TAOS-CI/issues/137_

----

--------
```bash
$ cat /var/log/apache2/error.log
./common/cibot_rest_api.sh: line 10: check_package: command not found
./common/cibot_rest_api.sh: line 11: check_package: command not found
./common/cibot_rest_api.sh: line 12: check_package: command not found
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M100  1954  100  1779  100   175   7658    753 --:--:-- --:--:-- --:--:--  7668
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M100  2122  100  1863  100   259   8364   1162 --:--:-- --:--:-- --:--:--  8391
```

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/git-bot-sec"><img src="https://github.sec.samsung.net/avatars/u/63289?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [git-bot-sec](https://github.sec.samsung.net/git-bot-sec)**
_Wednesday Jul 18, 2018 at 07:16 GMT_

----

:octocat: **cibot**: Thank you for posting issue #137. The person in charge will reply soon.

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Wednesday Jul 18, 2018 at 07:17 GMT_

----

* after PR https://github.sec.samsung.net/STAR/TAOS-CI/pull/125 version 6: 

```bash
root@AI2Quadro02:~# cat  /var/log/apache2/error.log
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1945  100  1770  100   175   6660    658 --:--:-- --:--:-- --:--:--  6654
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2099  100  1847  100   252   7395   1008 --:--:-- --:--:-- --:--:--  7417
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1954  100  1779  100   175   5018    493 --:--:-- --:--:-- --:--:--  5025
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2122  100  1863  100   259   7286   1012 --:--:-- --:--:-- --:--:--  7277
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1965  100  1862  100   103   3910    216 --:--:-- --:--:-- --:--:--  3903
```

**Comment by leemgs** [comment]
<a href="https://github.sec.samsung.net/geunsik-lim"><img src="https://github.sec.samsung.net/avatars/u/8170?" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [geunsik-lim](https://github.sec.samsung.net/geunsik-lim)**
_Thursday Jul 19, 2018 at 04:10 GMT_

----

Fixed by PR #125
